exodus_lambda/functions/origin_request.py

```diff
@@ -304,33 +319,9 @@ def validate_request(self, request):
             valid = False
         return valid
 
-    def handler(self, event, context):
-        # pylint: disable=unused-argument
-
-        request = event["Records"][0]["cf"]["request"]
-
-        if not self.validate_request(request):
-            return {"status": "400", "statusDescription": "Bad Request"}
-
-        self.logger.debug(
-            "Incoming request value for origin_request",
-            extra={"request": request},
-        )
-
-        request["uri"] = unquote(request["uri"])
-        original_uri = request["uri"]
-
-        if request["uri"].startswith("/_/cookie/"):
-            return self.handle_cookie_request(event)
-
-        uri = self.resolve_aliases(request["uri"])
-
-        listing_response = self.handle_listing_request(uri)
-        if listing_response:
-            self.set_cache_control(uri, listing_response)
-            return listing_response
-
-        table = self.conf["table"]["name"]
+    def handle_file_request(self, request, table, original_uri, uri):
+        # Try find the db entry corresponding to the uri. We try the uri, it's
+        # corresponding index path and then the fallback_uri.
```

Comment:
I think stating "and then the fallback_uri" is confusing here, because there's no concept of `fallback_uri` within this function.